roq-core/src/main/java/org/roqmessaging/core/QueueClientManager.java

```diff
@@ -1,176 +0,0 @@
-/**
```

Comment:
You should limit renaming files to a minimum because it makes it very hard to see the changes you've made.